GitUpdateScript.sh

```diff
@@ -94,26 +94,30 @@ rm Temp_dir_log #rm removes/dels the file
 }
 
 home_dir=$(pwd)  #v1 holds the home dir
-echo "Git Repositories Found:"
+
 
 # todo Count from find instead of iterating loop from file
-find -name .git| tee Temp_dir_log  #finds file with name .gitignore and stores results in file called FILENAME
 
-echo "Going to each git repositories and update:"
-while read file_dir
-do
 
-	GoToGitRepositoryDirectory
+arrayFileDir=($(find -name .git))
```

Comment:
instead of creating a file to hold find results, now using an array

Reply:
as discussed, moving this to a function is clean code